doc/dev/continuous-integration.rst

```diff
@@ -155,14 +155,25 @@ libboost
     ``ceph-libboost-*``, and they are instead installed into ``/opt/ceph``, so
     they don't interfere with the official ``libboost`` packages shipped by
     distro. Its build scripts are hosted at https://github.com/ceph/ceph-boost.
+    See https://github.com/ceph/ceph-boost/commit/2a8ae02932b2a1fd6a68072da8ca0df2b99b805c
```

Comment:
how about adding a note on how to use `dch` to add a changelog item, something like. 
```rst

to add a boilerplate changelog entry:

.. prompt:: bash $

   dch -v 1.80.0-1.1

```

Reply:
Shoot, I just missed this.  I actually edited the changelog manually :grimacing: